.github/workflows/generate_new_client_hermetic_build.yaml

```diff
@@ -0,0 +1,178 @@
+name: Generate new GAPIC client library (Hermetic Build)
+on:
+  workflow_dispatch:
+      # some inputs are ommited due to limit of 10 input arguments
+      inputs:
+        api_shortname:
+          required: true
+          type: string
+          description: "`api_shortname`: Name for the new directory name and (default) artifact name"
+        name_pretty:
+          required: true
+          type: string
+          description: "`name_pretty`: The human-friendly name that appears in README.md"
+        proto_path:
+          required: true
+          type: string
+          description: |
+            `proto_path`: Path to proto file from the root of the googleapis repository to the
+            directory that contains the proto files (without the version).
+            For example, to generate the library for 'google/maps/routing/v2',
+            then you specify this value as 'google/maps/routing'
+        product_docs:
+          required: true
+          type: string
+          description: "`product_docs`: Documentation URL that appears in README.md"
+        rest_docs:
+          required: false
+          type: string
+          description: |
+            `rest_docs`: If it exists, link to the REST Documentation for a service
+        rpc_docs:
+          required: false
+          type: string
+          description: |
+            `rpc_docs`: If it exists, link to the RPC Documentation for a service
+        api_description:
```

Comment:
nit: could you move this required field to the top (just below other required field).

Reply:
Done